pkg/cmd/pulumi/new.go

```diff
@@ -412,6 +425,21 @@ func newNewCmd() *cobra.Command {
 			if len(cliArgs) > 0 {
 				args.templateNameOrURL = cliArgs[0]
 			}
+			if args.listTemplates {
+				templates, err := getTemplates()
+				if err != nil {
+					logging.Warningf("could not list templates: %v", err)
+					return err
+				}
+				available, _ := templatesToOptionArrayAndMap(templates, true)
+				fmt.Println("")
```

Comment:
Why add an empty line? I don't think we do this in other commands (e.g. `pulumi stack ls`)